mmv1/templates/terraform/examples/cloudrunv2_job_sql.tf.erb

```diff
@@ -0,0 +1,71 @@
+resource "google_cloud_run_v2_job" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['cloud_run_job_name'] %>"
+  location = "us-central1"
+  launch_stage = "BETA"
+  
+  template {
+    template{
```

Comment:
```suggestion
    spec {
```
?

Reply:
This job resource is implemented using v2 API, which I believe `spec` got flattened and `template { template {}}` should be the correct format.